### PR TITLE
Add percents to Overview Item

### PR DESCRIPTION
**Pull request by kutsaniuk**
## Summary by CodeRabbit

- **New Features**
	- Introduced percentage display options in the Overview component, allowing users to show data as an upward or downward percentage.
- **Enhancements**
	- Added new configuration options for customizing the display of percentage values within overview items.

### CODE REVIEW

File: packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js [hunk 1]
```diff
@@ -5,6 +5,12 @@ export default {
   tags: ["autodocs"],
 };
 export const overviewItem = {
+  argTypes: {
+    percentType: {
+      control: "select",
+      options: ["up", "down"],
```
Comment (otaviolbarbosa) [inline packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js:11]: Is this a requirement? If not, up/down should be best to base on percent value (ie. +1% or -1%)

**Review by otaviolbarbosa — APPROVED** [review]

File: packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js (hunk 1 above)
Comment (kutsaniuk) [reply packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js:11]: No, it isn't. Thanks, I will change

File: packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js [hunk 2]
```diff
@@ -14,6 +14,8 @@ export const overviewItem = {
     isBig: false,
     isTransparent: false,
     isCard: false,
+    isPercent: false,
```
Comment (mykhailo-diakovych) [inline packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js:17]: You could delete `isPercent` props and check it if `percentValue !== 0`

Comment (mykhailo-diakovych) [reply packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js:17]: Screenshot 2023-12-21 at 14 23 33
Screenshot 2023-12-21 at 14 23 23

It is quite a same block with persentValue=0 and isPercent off/on

Comment (kutsaniuk) [reply packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js:17]: Goos, will fix, thanks!

File: packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js [hunk 3]
```diff
@@ -33,6 +34,7 @@ export const overviewItem = {
                 :is-big="args.isBig"
                 :is-transparent="args.isTransparent"
                 :is-card="args.isCard"
+                :percent-value="args.percentValue"
```
Comment (kewcoder) [inline packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js:37]: add new stories for percent variant

**Pull request merged**